src/tests/Microsoft.PowerFx.Interpreter.Tests/FlagTests.cs

```diff
@@ -46,5 +46,88 @@ public void Parser_DisableRowScopeDisambiguationSyntax(string expression)
 
             Assert.True(resultWithoutFlag.IsSuccess);
         }
+
+        [Theory]
+        [InlineData("X")]
+        [InlineData("$\"Test {X}\"")]
+        [InlineData("!X")]
+        [InlineData("First(Table(X))")]
+        [InlineData("X.Field1")]
+        [InlineData("X + 1")]
+        [InlineData("X + Date(2022, 10, 27)")]
+        [InlineData("X * 1")]
+        [InlineData("X < 2")]
+        [InlineData("\"The\" in X")]
+        [InlineData("123 in X")]
+        [InlineData("123 in Table(X)", false)]
+        [InlineData("IsBlank(X)")]
+        [InlineData("IsError(X)", false)]
+        [InlineData("Text(X)")]
+        [InlineData("Value(X)")]
+        [InlineData("Boolean(X)")]
+        [InlineData("Index([1,2,3].Value, X)")]
+
+        // Ensures expression binds without any errors - but issues a warning for the deferred(unknown) type.
+        public void DeferredTypeTest_EnableDeferredType(string script, bool isEvalAllowed = true)
```

Comment:
>DeferredTypeTest_EnableDeferredType [](http://example.com/codeflow?start=20&length=35)

what's relation of these tests to TypeTests?

Reply:
Ahh, I just missed to remove it while I was moving the test to different place. Removed now...